examples/hybrid/callbacks.jl

```diff
@@ -80,6 +95,47 @@ function affect_filter!(integrator)
     ODE.u_modified!(integrator, false)
 end
 
+function turb_conv_affect_filter!(integrator)
+    (; edmf_cache, Δt) = p
+    (;
+        edmf,
+        param_set,
+        aux,
+        case,
+        surf_params,
+    ) = edmf_cache
+    t = integrator.t
+    Y = integrator.u
```

Comment:
```suggestion
    (; t, Y) = integrator
```

Reply:
@charleskawczynski I think this is not working:
```
LoadError: type ODEIntegrator has no field Y
--
  | Stacktrace:
  | [1] getproperty
  | @ ./Base.jl:42 [inlined]
  | [2] turb_conv_affect_filter!(integrator::O

```

you can see in the buildkite stack

Reply:
Ah, yeah, just revert that